x-pack/plugins/monitoring/public/directives/alerts/index.js

```diff
@@ -74,36 +126,44 @@ const alertRowFactory = (scope, kbnUrl) => {
           />
         </KuiTableRowCell>
         <KuiTableRowCell tabIndex="0">
-          { linkToCategories[props.metadata.link] ? linkToCategories[props.metadata.link] : 'General' }
+          { linkToCategories[props.metadata.link] ? linkToCategories[props.metadata.link] :
+            props.intl.formatMessage({ id: 'xpack.monitoring.alerts.generalCategoryDescription', defaultMessage: 'General', }) }
         </KuiTableRowCell>
         <KuiTableRowCell tabIndex="0">
           { formatDateTimeLocal(props.update_timestamp) }
         </KuiTableRowCell>
         <KuiTableRowCell tabIndex="0">
-          { formatTimestampToDuration(props.timestamp, CALCULATE_DURATION_SINCE) } ago
+          <FormattedMessage
+            id="xpack.monitoring.alerts.trigeredAgoDescription"
```

Comment:
.trigeredAgoDescription -> .triggeredAgoDescription
Maybe miss letter 'g' in word triggered